framework/include/utils/RankTwoTensor.h

```diff
@@ -405,6 +405,14 @@ class RankTwoTensor
   /// returns this_ij * b_ijkl
   RankTwoTensor initialContraction(const RankFourTensor & b) const;
 
+
+  /// rank two permutation tensor (also known as Levi-Civita symbol):
+  /* eps(i1, i2, ..., iD) =
+  *   +1 if (i1, i2, ..., iD) is an even permutation of (0, 1, ..., D)
+  *   -1 if (i1, i2, ..., iD) is an odd permutation of (0, 1, ..., D)
+  /*    0 otherwise */
+  static int eps(unsigned int i, unsigned int j);
```

Comment:
Why does this have to be `static int`? The original permutation tensor include had only `int`.